packages/react-server/core/ReactServerAgent/Request.js

```diff
@@ -71,9 +74,14 @@ Request.prototype.query = function (queryParams) {
 
 Request.prototype.send = function (postParams) {
 	if (typeof postParams === 'undefined') {
-		return merge({}, this._postParams);
+		return merge({}, this._postParams || {});
+	}
+	if (postParams !== null) {
```

Comment:
FWIW, looking at the corresponding SuperAgent functions, they _always_ return `this`.  They are not trying to combine a setter with a getter function.